matscipy/calculators/polydisperse/calculator.py

```diff
@@ -0,0 +1,307 @@
+# ======================================================================
+# matscipy - Python materials science tools
+# https://github.com/libAtoms/matscipy
+#
+# Copyright (2014-2018) James Kermode, King's College London
+#                       Lars Pastewka, University of Freiburg
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# ======================================================================
+
+from __future__ import division
+
+import os
+
+import sys
+
+import time
+
+import numpy as np
+
+import ase
+
+from ase.calculators.calculator import Calculator
+
+from matscipy.neighbours import neighbour_list, first_neighbours
+
+try:
+    from scipy.special import factorial2
+    from scipy.sparse import bsr_matrix
+except ImportError:
+    warnings.warn('Warning: no scipy')
+
+###
+
+
+class IPL():
```

Comment:
Use more explicit names. Call this InversePowerLawPotential - keystrokes are cheap and abbreviations are difficult to understand for the casual reader.